app/lib/templates.dart

```diff
@@ -528,3 +538,8 @@ class PackageLinks extends PageLinks {
   @override
   String formatHref(int page) => '${_basePath ?? '/packages'}?page=$page';
 }
+
+abstract class LogoUrls {
+  static const String smallDartFavicon = '/static/favicon.ico';
+  static const String flutterLogo32x32 = '/static/img/flutter-logo-32x32.png';
```

Comment:
It is a bit confusing that we:
* have the two images in different extensions `.png` and `.ico`
* have the two images possibly in different resolutions

but do use them at the same places (see e.g. the `<link rel="short cut icon"` ...)